src/js/background/assignManager.js

```diff
@@ -274,6 +306,38 @@ const assignManager = {
     }
   },
 
+  async _onClickedBookmark(info) {
+
+    async function _getBookmarksFromInfo(info) {
+      const [bookmarkTreeNode] = await browser.bookmarks.get(info.bookmarkId);
+      if (bookmarkTreeNode.type === "folder") {
+        return await browser.bookmarks.getChildren(bookmarkTreeNode.id);
+      }
+      return [bookmarkTreeNode];
+    }
+
+    const bookmarks = await _getBookmarksFromInfo(info);
+    for (const bookmark of bookmarks) {
+      // Some checks on the urls from https://github.com/Rob--W/bookmark-container-tab/ thanks!
+      if ( !/^(javascript|place):/i.test(bookmark.url) && bookmark.type !== "folder") {
```

Comment:
Oh dang. This means nothing will happen when a user tries to open a `javascript:` or `place:` bookmark into a container, with no error message nor alt text telling them why they can't.

If we get bug reports about it, we could try moving this logic "up" into `init` before we add the listener to the context menu item.